src/Notes/MerchantEmailNotifications/MerchantEmailNotifications.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Handles merchant email notifications
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes\MerchantEmailNotifications;
+
+use Automattic\WooCommerce\Admin\Notes\Note;
+use Automattic\WooCommerce\Admin\Notes\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Merchant email notifications.
+ * This gets all non-sent notes type `email` and sends them.
+ */
+class MerchantEmailNotifications {
+	/**
+	 * Send all the notifications type `email`.
+	 */
+	public static function possible_send() {
```

Comment:
This could just be called `run()`

Reply:
Good idea! I renamed the method in the commit 79f811c